### PR TITLE
doc: add links to vlib doc - array and map

### PR DESCRIPTION
**Pull request by kahsa**
added links to vlib doc.

### CODE REVIEW

File: doc/docs.md [hunk 1]
```diff
@@ -1024,6 +1026,8 @@ There are further built-in methods for arrays:
 * `a.join(joiner)` concatenates an array of strings into one string
   using `joiner` string as a separator
 
+See all members of [array](https://modules.vlang.io/index.html#array)
```
Comment (spytheman) [inline doc/docs.md:1029]:
```suggestion
See all methods of [array](https://modules.vlang.io/index.html#array)
```

File: doc/docs.md [hunk 2]
```diff
@@ -641,6 +641,8 @@ age := 12
 println('age = $age')
 ```
 
+See all members of [string](https://modules.vlang.io/index.html#string)
```
Comment (spytheman) [inline doc/docs.md:644]:
```suggestion
See all methods of [string](https://modules.vlang.io/index.html#string)
```

File: doc/docs.md [hunk 3]
```diff
@@ -1300,6 +1304,8 @@ print(m)
 Maps are ordered by insertion, like dictionaries in Python. The order is a
 guaranteed language feature. This may change in the future.
 
+See all members of [map](https://modules.vlang.io/index.html#map)
```
Comment (spytheman) [inline doc/docs.md:1307]:
```suggestion
See all methods of [map](https://modules.vlang.io/index.html#map)
```

**Comment by kahsa** [comment]
Is it correct to use the word "all"?
Are there any other method docs written anywhere other than here?
Since we use these structures so often, we would like to append the information so that all public methods can be referenced.

**Pull request merged**